src/components/cards/SponsorCard.tsx

```diff
@@ -48,20 +53,20 @@ export default function SponsorCard({ sponsor }: SponsorCardProps) {
           />
         </View>
         <View style={CardStyles.headerTitle}>
-          <Text style={headingStyle}>{sponsor.sponsorName}</Text>
-          <Text style={primaryTextStyle}>{sponsor.email}</Text>
+          <Text style={headingStyle}>{sponsor.sponsor_name}</Text>
+          <Text style={primaryTextStyle}>{sponsor.description}</Text>
```

Comment:
There may need to be some amount of text formatting so that the description text doesnt run too close the the right border. I am also considering maybe not showing description at all expanding the sponsor tier badge similar to the "potential sponsor card" on the figma.

Reply:
> > I think the APISponsor and Sponsor interfaces do not have the same fields as the schema. The app successfully renders and acts as if it has the description and socials fields, but the build flags it...
> > ![image](https://user-images.githubusercontent.com/37993586/139622802-680ed81e-af8b-4282-b675-54361312c1fa.png)
> 
> You might have to add the type manually to the types file on our end so it wont be flagged

Even if I make the type locally, won't we still need to access certain fields of the schema (e.g. the description)? It's using those fields that the build flags.

Reply:
I'll have to hand the card's text formatting off to someone. I'm a little swamped atm

Reply:
Oh i definitely missed this comment, I'll take it from here